.buildkite/pipeline.yml

```diff
@@ -64,3 +64,28 @@ steps:
                   env:
                     - BUILDKITE_PULL_REQUEST=${BUILDKITE_PULL_REQUEST}
                     - GITHUB_PR_LABELS=${GITHUB_PR_LABELS}
+
+  - label: "Trigger x-pack/Heartbeat"
+    plugins:
+      - monorepo-diff#v1.0.1:
+          diff: "git diff --name-only HEAD~1"
```

Comment:
```suggestion
          diff: "git diff --name-only origin/${GITHUB_PR_TARGET_BRANCH}...HEAD"
```

Reply:
see https://github.com/elastic/beats/pull/38870/files ; for merge commits (e.g. against main) we want to run all the tests, so we'll like use a different, very simple, pipeline, details in https://github.com/elastic/ingest-dev/issues/3159#issuecomment-2049780190